src/automations/checkInactivity.js

```diff
@@ -102,19 +107,22 @@ async function scrapeInactiveIssues(client, references, issues) {
   const iterator = issues[Symbol.iterator]();
 
   for (let issue of iterator) {
+    const issueNumber = issue.number;
+    const repoName = issue.repository.name;
+    const repoOwner = issue.repository.owner.login;
+    const issueTag = `${repoName}/${issueNumber}`;
+    const repoTag = issue.repository.full_name;
+
     const inactiveLabel = issue.labels.find(label => {
       return label.name === client.cfg.activity.inactive;
     });
     if (inactiveLabel) continue;
+    if (references.has(issueTag) && !references.get(issueTag).review) continue;
```

Comment:
This statement is checking whether references has the issueTag AND whether the issueTag reference has a review property, which isn't what you want (all references already have the review property, also they aren't made in the first place). You're supposed to break out of the loop only if the reference doesn't exist.

Reply:
Doesn't the `references.get(issueTag).review` access `reviewedLabel`, which determines whether there is a 'reviewed' tag or not? From what I understand, the if block in question is checking whether references has the issueTag AND whether or not `reviewedLabel` is true, i.e there is a 'reviewed' label present. I think this checks whether there is a reference to the issue and if there is, then whether the PR is in 'needs review' state. If it is, then it jumps to next iteration.

If we break out of the loop only if the reference doesn't exist, then wouldn't the case where the reference exists, but the PR is in 'needs review' state be unhandled?